### ISSUE TITLE
[Feature]a folder system inside bookmarks

### ISSUE THREAD

**Issue by Robonau** [manga-download/hakuneko#1912]
**Is your feature request related to a problem? Please describe.**
a way to group your bookmarked manga would be nice having them all be in one place is kinda like putting everything on your desktop,it can get kinda cluttered/messy

**Describe the solution you'd like**
a way to group bookmarks together (like a folder)
(you could manually split daily/weekly/monthly/OnBreak or in to genres or anything really)

**Context (please complete the following information):**
 - Your current HakuNeko Version [e.g. 6.17]

**Additional context**
https://i.imgur.com/I9JEIYy.png

**Comment by Sheepux** [comment]
Had the same idea a few weeks ago.
I was also thinking putting the "completed" status so that my "check new content" feature would not scan those completed status.

For the genre/tags could be a sub section within the main status (or you could chose the order)
* daily
  - isekai
  - romance
* weekly
  - isekai
  - sport

and so on.
You could switch to
* isekai
  - daily
  - weekly

Also, i don't think that the daily/weekly is usefull when using the "new content detector feature" (but i'm the only one having it as it's not mature enougth).
But the choice to use it or not should be left to the user.

I'll probably take care of this on the 2021 release of hakuneko (global rework)

**Comment by 27136773** [comment]
I would suggest a tag based search as it is done for websites

**Comment by Sheepux** [comment]
Yes, but the way to show them using folders is also appealing to me (and that would be a user's choice).
Just have to find how we could let the user define the folder tags search by themselves with OR and AND

eg:  folder  adventure = tags: (adventure or fantasy or isekai+fantasy)

Yup... that's getting a bit complex :D I'll think about it.

**Comment by Robonau** [comment]
I agree that tag based search would probably be better for genres, but i still think that a folder system would be nice because it would allow you to organise however you like (genres and release frequency were just examples that i came up with on the spot)
